decidim-initiatives/spec/system/create_initiative_spec.rb

```diff
@@ -8,6 +8,20 @@
   let!(:authorization) { create(:authorization, user: authorized_user) }
   let(:login) { true }
 
+  let(:initiative_type_minimum_committee_members) { 2 }
+  let(:signature_type) { "any" }
+  let(:initiative_type_promoting_committee_enabled) { true }
+  let(:initiative_type) do
+    create(:initiatives_type,
+           organization: organization,
+           minimum_committee_members: initiative_type_minimum_committee_members,
+           promoting_committee_enabled: initiative_type_promoting_committee_enabled,
+           signature_type: signature_type)
+  end
+  let!(:other_initiative_type) { create(:initiatives_type, organization: organization) }
+  let!(:initiative_type_scope) { create(:initiatives_type_scope, type: initiative_type) }
+  let!(:other_initiative_type_scope) { create(:initiatives_type_scope, type: initiative_type) }
+
```

Comment:
These lines weren't modified on the original PR #9928. Are you sure they are necessary? I wanted to run this spec with these files removed locally but I have some situation with our good old friend Webpacker 😞 

I see the same is happening in #10741. 

Is this change coming from a git merge conflict resolution?

Reply:
This change is added on purpose. This file has been changed since version 0.26, and just applying the original patch ... caused a lot of specs to fail. 
Please see the commit added: https://github.com/decidim/decidim/commit/680158f43cce2241e13b0e63ecb9c748f3442e9c
Please see the below run for details: https://github.com/decidim/decidim/actions/runs/4746210829/jobs/8429548974

Reply:
Oh, I missed that because you've force pushed multiple times, so this run wasn't visible for me. Thanks for mentioning it!

On this case what I'd prefer is to have two commits: 

1. the original backport commit with the failure of the CI
2. the explicit fix with the manual backport/change that you needed for this to work. Sometimes you may need to try multiple changes and you can't try them out locally (as you may need to regenerate the test_app just for running a spec and that could be time consuming, specially if working with multiple backports and versions as we usually do). On these cases, I use the `git commit --amend --no-edit` and `git push --force` so I only have 2 commits and the git history is clean.

If you have any other proposal please share it. For now this LGTM as is, so I'll merge it.